sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -260,9 +260,7 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase with ExecutionE
   def invalidFractionOfSecondError(): DateTimeException = {
```

Comment:
Let's pass the actual invalid fraction here, and show it to users.

Reply:
Agreed, will fix.